### PR TITLE
Add sams blog

### PR DESCRIPTION
**Pull request by sw00d**
Blog time baby! Check her out and feel free to give me critique wherever necessary!

### CODE REVIEW

File: _posts/2020-11-22-Websockets-and-the-frontend.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+---
+
+There are several techniques to wield the power of WebSockets. The method we chose for this project was fairly simple: we use a standard REST API to make changes on the backend, then we send a message through to the WebSocket to indicate specificly what data has changed. Furthermore, to make changes, we can call a PATCH/PUT/POST method to the API, and once that resolves, we send a message up to the WebSocket. This message is then sent to the other clients that are listening. To reiterate, the message dictates what data needs updated. So once a client receives a message from it’s WebSocket connection, it triggers a GET request to fetch the newly updated data. This is a, somewhat, nontraditional way to set up a WebSocket and definitely has its cons. It would be more efficient and responsive for the client to send/receive data straight through the socket’s connection instead of pairing it with a REST architecture. While we may change this in the future, we were going for simplicity in this initial stage. WebSockets are fairly new to the team so having a simple implementation helps new dev's jump on the project and understand it quicker.
```
Comment (Logan-Ruf) [inline _posts/2020-11-22-Websockets-and-the-frontend.md]: "WebSockets are fairly new to the team so having a simple implementation helps new devs jump on the project and understand it quicker." Hey! Don't throw me under the bus lol! I would rephrase this as if you are making it easier for an inexperienced reader. Don't want a client to read this and think we aren't experienced.

File: _posts/2020-11-22-Websockets-and-the-frontend.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+# Let's get to work
+
+I recently implemented our existing WebSocket for CKC Management into a barebones React Native app. Our web app is built with Vue.js but I’ll be going over the React Native implementation. We are assuming that you already have a backend running at this point that I will not be covered in this article. 
+
+This whole thing is fairly straightforward but I assume you have a React Native project up and running at this point. 
+
+To start, we need to initialize the connection to our WebSocket:
```
Comment (Logan-Ruf) [inline _posts/2020-11-22-Websockets-and-the-frontend.md:28]: Maybe link something here for users who don't know how to set-up a basic React project

**Review by Logan-Ruf — CHANGES_REQUESTED** [review]
Did you re-encode your gif after capturing it? Seems a little large for how short it is.

File: _posts/2020-11-22-Websockets-and-the-frontend.md (hunk 1 above)
Comment (sw00d) [reply _posts/2020-11-22-Websockets-and-the-frontend.md]: Good call. I just knew Bailey built it cause he hadn't worked in depth with them before but you're totally right. Will rephrase.

File: _posts/2020-11-22-Websockets-and-the-frontend.md (hunk 2 above)
Comment (sw00d) [reply _posts/2020-11-22-Websockets-and-the-frontend.md:28]: Oh yes I forgot that. Thanks

**Review by Logan-Ruf — APPROVED** [review]
The gif think probably won't matter much
